sklearn/linear_model/_sgd_fast.pyx

```diff
@@ -629,10 +630,11 @@ def _plain_sgd(cnp.ndarray[double, ndim=1, mode='c'] weights,
             # report epoch information
             if verbose > 0:
                 with gil:
+
```

Comment:
This diff is unrelated. Can you revert? (This makes `git-blame` easier to look through)